ACM_General/core/templates/core/base.html

```diff
@@ -1,85 +1,52 @@
-<!DOCTYPE html>
-{% load static %}
-
-{% block html_tag %}<html lang='en'>{% endblock %}
+<!DOCTYPE html>{% load static %}
+{% block html_tag %}
+  <html lang='en'>{% endblock %}
```

Comment:
Looks like we have two options, first would be to use some **middleware** (namely [django-htmlmin](https://pypi.python.org/pypi/django-htmlmin)), and the second would be **spaceless tags**:
```
{% spaceless %}
{% endspaceless %}
```